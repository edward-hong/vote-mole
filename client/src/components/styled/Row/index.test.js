import React from 'react'
import { render } from '@testing-library/react'

import Row from '.'

describe('Row', () => {
	it('renders correctly', () => {
		const { container } = render(<Row />)

		expect(container.firstChild).toMatchSnapshot()
	})
})
