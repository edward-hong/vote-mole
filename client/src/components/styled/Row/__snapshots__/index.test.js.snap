// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row renders correctly 1`] = `
.c0 {
  margin-top: 1.5rem;
}

<div
  class="c0 row"
/>
`;
